client/jetpack-connect/site-url-input.jsx

```diff
@@ -119,7 +121,7 @@ class JetpackConnectSiteUrlInput extends PureComponent {
 						ref={ this.refInput }
 						id="siteUrl"
 						autoCapitalize="off"
-						autoFocus="autofocus"
```

Comment:
Hi @keoshi,

I get the same effect with just removing line 122 

```html
autoFocus="autofocus"
```

Is there a place where autoFocus is really needed for this component? I only could find site-url-input used in client/jetpack-connect/main.jsx and couldn't see why autoFocus would be needed in there.

I hope my comment is not way too off, as I'm new to the codebase :)

Reply:
@omidfi, yes, autofocusing this field is expected and needed in the Jetpack Connect flow, for example in https://wordpress.com/jetpack/connect/